ui-v2/package.json

```diff
@@ -34,7 +34,7 @@
     ]
   },
   "devDependencies": {
-    "@hashicorp/consul-api-double": "^1.5.3",
+    "@hashicorp/consul-api-double": "^2.0.1",
```

Comment:
A major version bump? What changed? Do you PR to this repo? Do the changes get code reviewed?

Reply:
> A major version bump? What changed?

Yup, major. I changed the name of one of the cookies you can use to debug/test from `CONSUL_ENABLE_ACLS` to `CONSUL_ACLS_ENABLE` to bring it in line with all of the other variables I use to debug and test the UI (they all follow `CONSUL_<MODEL_NAME>_<SETTING>` now). So, unfortunately breaking yes. I'm the only person that I know of using this so I'd say it's not a big deal in the scheme of things, but you never know, plus its habit I suppose. Breaking means major bump.

> Do you PR to this repo? Do the changes get code reviewed?

No I don't PR changes, I commit straight onto `master` and publish to `npm` to release, if there is a 'bug' I release a patch version, but generally it's reasonably well tested first considering what it is. This repo is only my 'api double' _templates_. They don't really do anything, it's only a templated version of the `consul` HTTP API. So its pretty much a tonne of JSON text files.

Also, the module isn't shipping code (in the Consul UI), it's only to make it easier for me to test using massive amounts of data.

I sometimes ask the backend team to review the code if it's for a new upcoming API that hasn't been nailed down yet to make sure I can build out the UI without the backend being built.

If you want to help out with `api-double` (the actual template renderer, not the specific `consul-api-double` templates) I'd be more than happy for you to pitch in if that's what you want to do. I've mention people helping out a few times but up until now no-one seems interested? Let me know, happy to walk you through.

Reply:
I just noticed this whilst I was searching for the 'DDAU alternative' PR, thought it gives some good context `api-double` vs `consul-api-double` so posting here for reference:

https://github.com/hashicorp/consul/pull/4190#discussion_r193176956